File: striver/S3L1/remove_duplicates_sorted_array.py
class Solution(object):
    def removeDuplicates(self, nums):
        fr = 0
        sr = 0
        while fr<nums.length:
            if nums[sr] != nums[fr]:
                sr+=1
                nums[sr] = nums[fr]
            fr+=1
        return sr+1
        

File: striver/S3L1/remove_duplicates_sorted_array.java
package S3L1;

public class remove_duplicates_sorted_array {
    public int removeDuplicates(int[] nums) {
        int fr = 0;
        int sr = 0;
        while(fr<nums.length){
            if(nums[sr] != nums[fr]) {
                nums[++sr] = nums[fr];
            }
            fr++;
        }
        return ++sr;
    }
}
